docs/development.md

```diff
@@ -32,32 +32,21 @@ TruffleSqueak checkout.
 Using the `trufflesqueak-jvm` environment file ensures that the Graal compiler
 is built with TruffleSqueak and enabled by default.
 For this, however, you must use a JVMCI-enabled JDK, such as the
-[JDK11-based LabsJDK][labsjdk].
-The environment file also instructs mx to build an installable component for the
-GraalVM Updater, which can be installed with:
-
-```bash
-$GRAALVM_HOME/bin/gu install -f -L \
-   "$(../mx/mx --env trufflesqueak-svm paths SMALLTALK_INSTALLABLE_SVM_JAVA11)"
-```
+[JDK11-based LabsJDK][labsjdk]. Use `mx fetch-sdk` to download the LabsJDK.
 
 ## Development With Other GraalVM Languages
 
-If you need access to other GraalVM languages during development, there are two options:
+If you need access to other GraalVM languages during development, there are one option:
 
-First, you can download the latest [GraalVM dev build][graalvm_dev_build] and install additional GraalVM languages and components using [`gu`][gu] (e.g., `gu install python`). To install TruffleSqueak, follow these steps:
-
-1. Build TruffleSqueak via `mx --env trufflesqueak-jvm build`
-2. Run `mx --env trufflesqueak-jvm paths SMALLTALK_INSTALLABLE_JAVA11` to find the path to the installable
-3. Install it via `$GRAALVM_HOME/bin/gu install -f -L path/to/the/trufflesqueak-installable.jar`
-4. Run `$GRAALVM_HOME/bin/trufflesqueak`
-
-Or second, you can build other languages from source. For this, please refer to the [instructions for building custom GraalVM distributions][graalvm_vm_readme]. As an example, here is how to build a GraalVM with the Graal compiler, TruffleSqueak, the GraalVM Updater, and GraalVM's Node.js runtime:
+You build TruffleSqueak and other languages from source. For this, please refer to the [instructions for building custom GraalVM distributions][graalvm_vm_readme]. As an example, here is how to build a GraalVM with the Graal compiler, TruffleSqueak and GraalVM's Node.js runtime:
 
 ```bash
-cd trufflesqueak/../graal/vm
+cd trufflesqueak-build/
+git clone git@github.com:oracle/graaljs.git # Clone graal language repository
+cd graal/vm # Change to the GraalVM folder
 mx --dy trufflesqueak,/graal-nodejs,/compiler build
-mx --dy trufflesqueak,/graal-nodejs,/compiler graalvm-home # print path to $GRAALVM_HOME
+export GRAALVM_HOME=$(mx --dy trufflesqueak,/graal-nodejs,/compiler graalvm-home)
```

Comment:
When trying out GraalPy we had issues with differing BouncyCastle library versions, which (at least) I fixed by locally changing the BouncyCastle related stuff in `mx.trufflesqueak/suite.py`. @antonykamp is this also sth. we should add here? 🙂